environment.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
   - fiona
   - esmpy!=8.1.0  # see github.com/ESMValGroup/ESMValCore/issues/1208
   - geopy
-  - iris>=3.1.0
+  - iris>=3.1.0,!=3.2.0.post0,!=3.2.0
```

Comment:
why pin on the more general (and possibly functional, if they release a bugfix but still keep the minor version, and the micro version) 3.2.0

Reply:
Because that version already exists and contains the bug (just tested).

```bash
> mamba list iris
# packages in environment at /mnt/lustre02/work/bd0854/b309141/mambaforge/envs/yyy:
#
# Name                    Version                   Build  Channel
iris                      3.2.0              pyhd8ed1ab_0    conda-forge
```

Reply:
ah OK, but we must remember this might have to change if they fix it and it's still `3.2.0.something`

Reply:
The pin `!=3.2.0` still allows `3.2.0.post0`, so I guess we should be fine here

Reply:
so they did find a bug since they did a post a few hours after the main one, I thought they went straight to post for some reason